ams/ams-server/src/main/java/com/netease/arctic/ams/server/ArcticMetaStore.java

```diff
@@ -317,6 +356,32 @@ private static void startSyncDDl(final TServer server) {
     t.start();
   }
 
+  private static void checkLeader() {
+    Thread t = new Thread(() -> {
+      while (true) {
+        try {
+          Thread.sleep(2 * 1000);
+        } catch (InterruptedException e) {
+          LOG.warn("notLeader thread was interrupted: " + e.getMessage());
+        }
+        try {
+          if (haService != null && isLeader.get() &&
+              !haService.getMaster().equals(haService.getNodeInfo(conf.getString(ArcticMetaStoreConf.THRIFT_BIND_HOST),
```

Comment:
so, Ams instances should have different ArcticMetaStoreConf.THRIFT_BIND_HOST configuration?   

`(haService != null && isLeader.get() &&
              !haService.getMaster().equals(haService.getNodeInfo(conf.getString(ArcticMetaStoreConf.THRIFT_BIND_HOST),conf.getInteger(ArcticMetaStoreConf.THRIFT_BIND_PORT)`

Reply:
support host prefix in new commit, please take a look